content/en/dashboards/reporting.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Reporting
+kind: documentation
+disable_toc: false
+is_public: true
+beta: true
+is_beta: true
+further_reading:
+  - link: "https://www.datadoghq.com/blog/template-variable-associated-values/"
+    tag: "Blog"
+    text: "Use associated template variables to refine your dashboards"
+  - link: "https://learn.datadoghq.com/course/view.php?id=8"
+    tag: "Learning Center"
+    text: "Building Better Dashboards"
+---
+
+<div class="alert alert-warning">This feature is currently in private beta. To request access, reach out to support@datadoghq.com.</div>
+
+## Overview
+
+Reporting automatically sends a visual summary of a dashboard to selected recipients on a schedule. The generated reports display images of widgets from a dashboard in a linear HTML format.
+
+{{< img src="dashboards/reporting/report_example.png" alt="An example report titled Checkout KPI Report showing a date, a description, a dashboard link, and 3 images of dashboard widgets" style="width:100%;" >}}
+
+## Schedule a report
+
+Create a new report from any dashboard by opening the dashboard’s cog menu and selecting **Schedule a Report**.
```

Comment:
Regarding **any dashboard**:

- This feature is only supported for timeboards.
  - Does that mean we should use the word "timeboard"? I'm not sure -- there are 3 types of custom dashboard, and while 2 of them _(the non-screenboard options)_ are internally considered timeboards, only 1 is explicitly presented to users as such. See [here](https://docs.datadoghq.com/dashboards/) for an example of what I mean.
- We support it for custom timeboards and integration timeboards, but currently are only planning on GAing the custom timeboard support. So I think it would be best to specify **custom** here.

Reply:
By the way, the documentation page linked above has an error on it: _"They are commonly used as status boards or storytelling view, which update in real-time and can represent fixed points in the past."_

Reply:
Thanks for confirming those details. From a docs perspective, I think it's best to err on the side of maximum accuracy, so I've updated this to "any custom timeboard".
Also, regarding the page linked above, I've opened the following PR to address this:
https://github.com/DataDog/documentation/pull/14566
Thanks for pointing it out!